Chaincase.Common/Global.cs

```diff
@@ -260,6 +246,8 @@ public async Task InitializeNoWalletAsync(CancellationToken cancellationToken =
                 #endregion SynchronizerInitialization
 
                 TransactionBroadcaster = new TransactionBroadcaster(Network, BitcoinStore, Synchronizer, Nodes, WalletManager, null);
```

Comment:
this can also be a separate service?

Reply:
I'm not sure I follow. Because it relies on Synchronizer and Nodes, which get built during `InitializeNoWalletAsync` I'm not sure it can be part of the service recruitment/configureDI method. What do you mean? Do you think this should live outside of what becomes BlazorApp/ChaincaseLifecycle or it should belong to DI or something else?

Reply:
Problem: `Nodes`, a `TransactionBroadcaster` dependency is more of an entity than a service. Especially because it requires `AddressManager` to be loaded from a file and passed as a connectionParameter before it is constructed I think that makes `TransactionBroadcaster` an entity that needs to be managed by the lifecycle object as well.

Perhaps it could be swapped out for a TransactionBroadcastService if it got `(NBitcoin.NodesGroup)Nodes`  later on or as part of a service